hasura/migrations/default/1705087600730_alter_table_app_drop_metadata/up.sql

```diff
@@ -0,0 +1,11 @@
+alter table "public"."app"
```

Comment:
destructive in production. we first need to create a migration that moves all existing values to `app_matadata`, then in a separate PR we incorporate this migration